runtime/src/snapshot_utils.rs

```diff
@@ -1930,6 +1920,25 @@ pub fn rebuild_storages_from_snapshot_dir(
     let accounts_hardlinks = bank_snapshot_dir.join(SNAPSHOT_ACCOUNTS_HARDLINKS);
     let account_run_paths: HashSet<_> = HashSet::from_iter(account_paths);
 
+    // Read out the obsolete accounts if fastboot_version is >= 2
```

Comment:
Can this comment indicate _why_ version >= 2 is important? IMO the _what_ is less useful.